docs/docs/reference/metaprogramming/macros.md

```diff
@@ -78,7 +75,7 @@ The type signatures of quotes and splices can be described using
 two fundamental types:
 
 - `Expr[T]`: abstract syntax trees representing expressions of type `T`
-- `Type[T]`: type structures representing type `T`.
+- `Type[T]`: non erased representation of type `T`.
```

Comment:
Above in line 48 you say "type structure"

Reply:
Yes, there is much more to improve in this document. Might need a full rewrite.